Dockerfile.dev

```diff
@@ -50,8 +50,5 @@ RUN SETUPTOOLS_SCM_PRETEND_VERSION_FOR_FLYTEKIT=$PSEUDO_VERSION \
     && chown flytekit: /home \
     && :
 
-
-ENV PYTHONPATH="/flytekit:/flytekit/plugins/flytekit-k8s-pod:/flytekit/plugins/flytekit-deck-standard:"
```

Comment:
removing this broke [flyteremote integration tests](https://github.com/flyteorg/flytekit/blob/master/tests/flytekit/integration/remote/test_remote.py) due to the way tasks+workflows are [registered](https://github.com/flyteorg/flytekit/blob/master/tests/flytekit/integration/remote/test_remote.py#L38-L55). The tests end up present in the image under the `tests` module, which lives under the `flytekit` directory and since we had `/flytekit` as part of `PYTHONPATH` the task was able to be run once `pyflyte-execute` ran.

Reply:
fix in https://github.com/flyteorg/flytekit/pull/2701.